modules/custom_operations/tests/requirements.txt

```diff
@@ -0,0 +1,5 @@
+torch==1.8.1
+torchvision==0.9.1
+open3d==0.14.1
```

Comment:
Why is this needed?
Also, not sure about tensorboard

Reply:
both `open3d` and `tensorboard` are needed to test (and show how to support) `SparseConv` operation
https://github.com/likholat/openvino_contrib/blob/197109ccff1c2dc9770e09ce85cc2a25754e6faf/modules/custom_operations/examples/sparse_conv/export_model.py#L9